ui/app/send.js

```diff
@@ -175,24 +174,61 @@ SendTransactionScreen.prototype.render = function () {
             'To:',
           ]),
 
-          h(EnsInput, {
+          h('input.large-input.send-screen-input', {
             name: 'address',
-            placeholder: 'Recipient Address',
-            onChange: () => {
+            list: 'addresses',
+            placeholder: 'Address',
+            value: this.state.newTx.to,
+            onChange: (event) => {
               console.log('event', event.target.value)
               this.setState({
-                newTx: Object.assign(
-                  this.state.newTx,
-                  {
-                    to: event.target.value,
-                  }
-                ),
+                newTx: {
+                  ...this.state.newTx,
+                  to: event.target.value,
+                },
               })
             },
-            network,
-            identities,
-            addressBook,
-          }),
+          }, [
+          ]),
+
+          h('datalist#addresses', {}, [
+            // Corresponds to the addresses owned.
+            Object.keys(props.identities).map((key) => {
```

Comment:
Try [Object.entries](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries)

You can do:
```
Object.entries(props.identities).map(([key, {address, name}]) => ...)
```